PWGJE/Core/JetTaggingUtilities.h

```diff
@@ -44,10 +44,59 @@ enum JetTaggingSpecies {
   gluon = 5
 };
 
+enum BJetTaggingMethod {
+  IPsN1 = 0,
+  IPsN2 = 1,
+  IPsN3 = 2,
+  IPs3DN1 = 3,
+  IPs3DN2 = 4,
+  IPs3DN3 = 5,
+  SV = 6,
+  SV3D = 7
+};
+
 namespace jettaggingutilities
 {
 const int cmTomum = 10000; // using cm -> #mum for impact parameter (dca)
 
+struct BJetParams {
+  float mJetpT = 0.0;
+  float mJetEta = 0.0;
+  float mJetPhi = 0.0;
+  int mNTracks = -1;
+  int mNSV = -1;
+  float mJetMass = 0.0;
+};
+
+struct BJetTrackParams {
+  double mTrackpT = 0.0;
+  double mTrackEta = 0.0;
+  double mDotProdTrackJet = 0.0;
+  double mDotProdTrackJetOverJet = 0.0;
+  double mDeltaRJetTrack = 0.0;
+  double mSignedIP2D = 0.0;
+  double mSignedIP2DSign = 0.0;
+  double mSignedIP3D = 0.0;
+  double mSignedIP3DSign = 0.0;
+  double mMomFraction = 0.0;
+  double mDeltaRTrackVertex = 0.0;
+};
+
+struct BJetSVParams {
+  double mSVpT = 0.0;
+  double mDeltaRSVJet = 0.0;
+  double mSVMass = 0.0;
+  double mSVfE = 0.0;
+  double mIPXY = 0.0;
+  double mCPA = 0.0;
+  double mChi2PCA = 0.0;
+  double mDispersion = 0.0;
+  double mDecayLength2D = 0.0;
+  double mDecayLength2DError = 0.0;
+  double mDecayLength3D = 0.0;
+  double mDecayLength3DError = 0.0;
+};
+
 //________________________________________________________________________
```

Comment:
in Lines 103 and 115 swap the numbers for pdg keywords where possible

Reply:
I've changed them I could